apps/frontend-pwa/src/pages/course/[courseId]/element/[id].tsx

```diff
@@ -42,14 +56,18 @@ const ORDER_TYPE_LABEL = {
 interface Props {
   courseId: string
   id: string
+  description: string
+  submitText: string
```

Comment:
Neither description nor submitText are provided to the page through the props function at the end of the file. Therefore, these values either have to be removed or fetched from somewhere else.

Reply:
removed unused arguments